requirements/requirements-python3.7.txt

```diff
@@ -115,7 +115,6 @@ elasticsearch-dsl==7.2.0
 elasticsearch==7.5.1
 email-validator==1.1.1
 entrypoints==0.3
-enum34==1.1.10
```

Comment:
Where'd this go?

Reply:
Same here - I do no know, but I also do not care now - since all the generation is done automated and we have tests confirming that this set of requirements work fine.

Reply:
Curious, this should have gone as soon as you merged #8648 (Apri 30, 2020), as this dep was removed in #5146 (Apr 22, 2019!)

Anyway, yeah cool.